packages/gatsby-theme-api-docs/README.md

```diff
@@ -67,6 +67,28 @@ The file location determines the `apiKey` through which it can be addressed:
 - `src/api-specs/bar/api.raml` -> apiKey `bar`
 - `src/api-specs/baz/baz.raml` -> apiKey `baz`
 
+### Overriding Common Types
+
+If you want to override common types, create a file called `type-locations.yaml` under the `src/data` folder of the website. Use the following format to override the types you want:
```

Comment:
```suggestion
If you want to override the location to link to for certain API data types, create a file called `type-locations.yaml` under the `src/data` folder of the website. This can be useful to choose manual markdown based documentation for certain types or to link ot shared types used on other websites like public standards. Use the following format to override the type locations:
```